CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ The stress tool `influx_stress` will be removed in a subsequent release. We reco
 - [#7585](https://github.com/influxdata/influxdb/pull/7585): Return Error instead of panic when decoding point values.
 - [#7812](https://github.com/influxdata/influxdb/issues/7812): Fix slice out of bounds panic when pruning shard groups. Thanks @vladlopes
 - [#7822](https://github.com/influxdata/influxdb/issues/7822): Drop database will delete /influxdb/data directory
+- [#7835](https://github.com/influxdata/influxdb/pull/7835): Bind backup and restore port to localhost by default
```

Comment:
Needs to be moved to 1.3 section now.

Also, I think we need to add a note under a configuration changes (like prior releases) that nodes the change in default.  Some people _may_ be relying on the current default for remote backups.